sessionctx/variable/sysvar.go

```diff
@@ -2518,6 +2520,59 @@ var defaultSysVars = []*SysVar{
 		s.PlanCacheInvalidationOnFreshStats = TiDBOptOn(val)
 		return nil
 	}},
+	// runtime filter variables group
+	{Scope: ScopeGlobal | ScopeSession, Name: EnableRuntimeFilter, Value: BoolToOnOff(DefEnableRuntimeFilter), Type: TypeBool,
+		SetSession: func(s *SessionVars, val string) error {
+			s.enableRuntimeFilter = TiDBOptOn(val)
+			return nil
+		},
+		GetSession: func(s *SessionVars) (string, error) {
+			return BoolToOnOff(s.enableRuntimeFilter), nil
+		},
+	},
+	{Scope: ScopeGlobal | ScopeSession, Name: RuntimeFilterTypeName, Value: DefRuntimeFilterType, Type: TypeStr,
```

Comment:
Not a big problem and I think it's OK to remain them unchanged.
Just to mention, usually, we don't implement `GetSession` and in that case, tidb will return what the user inputs.

Reply:
RuntimeFilter has different type "IN,MIN_MAX,BLOOMFILTER". Different types of queries are suitable for different types. So we need to change the type in session var.

Reply:
Also, I removed get session function